cmd/xcode.go

```diff
@@ -151,7 +162,18 @@ func scanXcodeProject(cmd *cobra.Command, args []string) error {
 		return ArchiveError{toolXcode, err.Error()}
 	}
 
-	certsUploaded, provProfilesUploaded, err := codesigndoc.ExportCodesignFiles(archivePath, absExportOutputDirPath, certificatesOnly, isAskForPassword)
+	certificatesToExport, profilesToExport, err := codesigndoc.CollectCodesignFiles(archivePath, certificatesOnly)
+	if err != nil {
+		return err
+	}
+	certsUploaded, provProfilesUploaded, err := codesign.UploadAndWriteCodesignFiles(certificatesToExport,
```

Comment:
pls leave a comment that certificatesOnly will lead to 0 elements certsUploaded, that's why no need to pass it further